<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="auto_front">
     <property name="text">
      <string>Autostart on front</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="auto_back">
     <property name="text">
      <string>Autostart on back</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="loop">
     <property name="text">
      <string>Loop playback</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mute">
     <property name="text">
      <string>Mute audio</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="dimension_grid">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="height_lbl">
       <property name="text">
        <string>Height (px, -1 for auto)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSpinBox" name="height">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>4000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="width">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>4000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="width_lbl">
       <property name="text">
        <string>Width (px, -1 for auto)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
